existing-service/src/main/java/de/evoila/cf/cpi/existing/ExistingServiceFactory.java

```diff
@@ -76,7 +78,7 @@ public ServiceInstance preCreateInstance(ServiceInstance serviceInstance, Plan p
 
     @Override
     public ServiceInstance postCreateInstance(ServiceInstance serviceInstance, Plan plan) throws PlatformException {
-        serviceInstance.setHosts(existingEndpointBean.getHosts());
+        serviceInstance.setHosts(existingEndpoints.getEndpoints().get(0).getHosts());
```

Comment:
This is not good practise. I think we agreed on that you have a field in plan which is actually defining which existing Endpoint to use? In Plan there is already a field called `private String endpointName;`. 

So please rewrite this to:
```java
List<ServerAddress> hosts = existingEndpoints.getEndpoints().steam().filter(e -> {
  if (e.getName.equals(plan.getMetadata().getEndpointName()) 
    return true;
  else
   return false;
});
```

Reply:
done. Can you please check?